README.md

```diff
@@ -128,6 +133,7 @@ There are also some conventions that must be followed:
     - The string should explain why the error would occur.
     - A particular track's implementation of the exercise **need not** necessarily check that the error includes that exact string as the cause, depending on what is idiomatic in the language (it may not be idiomatic to check strings for error messages).
   - Test cases that only some tracks should implement, for example because it would unnecessarily increase the complexity of the exercise in some but not all languages, mark it with an `optional`-key. Multiple cases related to the same reason for optionality should have the same key. The decision that a test case is optional will often be made in the PR discussion, so don't worry about it too much while creating a PR.
+  - Each test case must have a unique UUID specified in its `"uuid"` key. A UUID can be randomly generated using the [UUID Generator](https://www.uuidgenerator.net/version4).
```

Comment:
Why not `configlet uuid` instead?

Reply:
My reasoning was that `configlet` is harder to get setup locally. One has to download first, whereas using a website is easier. I can add a second note about using `configlet`?

Reply:
But the other doc says that we will require `configlet` for formatting anyway, so that setup cost is already there.

Reply:
> But the other doc says that we will require `configlet` for formatting anyway, so that setup cost is already there.

True, although we won't get there initially. `configlet` will be updated sometime later, so I think I'll leave the current comment in and then update it later once we've updated `configlet`.

Reply:
Presumably you added the UUIDs via a script, right? Could you maybe share it so that one can run `./add-uuids` to just deal with it as another alternative until it's properly automated?

Reply:
I added it with an F# script, so that's probably not ideal. That said, I _am_ working on tooling to automatically add missing uuids. I hope to finish this somewhere this week or the next. I'll then update the instructions.